README.md

```diff
@@ -1,55 +1,67 @@
+
+### Double Opt-In
+
+In order to verify the email for a new subscription:
 
-- Email Alert Signup: a page displaying the title of what the User is signing up to
-- Tags: A key and array value pair which when are used by the [Email Alert API](https://github.com/alphagov/email-alert-api) to forward the user onto the signup URL
+- We [send a verification email using email-alert-api](https://github.com/alphagov/email-alert-frontend/pull/629)
+- The email [contains a link with a unique token for the subscription](https://github.com/alphagov/email-alert-api/pull/1051)\*
+- Clicking on the link [completes the signup process](https://github.com/alphagov/email-alert-frontend/pull/615)
 
-## Dependences
+\* Note that [the token strategy has changed over time](https://github.com/alphagov/email-alert-api/pull/1083).
 
-- [alphagov/email-alert-api](https://github.com/alphagov/email-alert-api)
-- [alphagov/content-store](https://github.com/alphagov/content-store)
-- [alphagov/static](https://github.com/alphagov/static)
+### Manage
 
-## Running the application
+This allows the user to list, modify and delete their subscriptions [[login](https://www.gov.uk/email/manage/authenticate)]. It uses a similar, but separate email/token process to authenticate a user, establishing a session for them to make their changes.
+
+## Nomenclature
+
+### Tags and links
+
+Uniquely define a list people can subscribe to. The criteria within are used to figure out whether an update is relevant subscribers in the list. Defined in [the docs for email-alert-api](https://github.com/alphagov/email-alert-api/blob/master/doc/matching-content-to-subscriber-lists.md).
 
 ```
-  $ ./startup.sh
+tags: { topics: { any: ["business-tax/vat"] } }
 ```
 
-If you are using the GDS development virtual machine then the application will be available on the host at http://email-alert-frontend.dev.gov.uk/
+```
+links: { topics: { any: ["1ddaaacb-0981-4abc-a532-a5111f2bea6b"] } }
+```
 
-## Running the test suite
+### Tags and links (again!)
+
+These terms are also used by [the legacy `email_alert_signup` schema](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/email_alert_signup.jsonnet). Note that the flat lists of base paths or Content IDs are interpreted as "any" in the above definition.
 
 ```
-$ bundle exec rake
+"tags": { "countries": ["foreign-travel-advice/canada"] }
 ```
 
-## Content Item
+```
+"links": { "countries": ["f402b8de-2e99-4ff3-949f-31fe65796cae"] }
+```
 
-The [schema](https://github.com/alphagov/govuk-content-schemas/tree/master/formats/email_alert_signup) sets out the fields required for the Content Item. The only two fields required in the details hash are `summary` and `tags`:
+## Running the test suite
```

Comment:
Is this "running the test suite" section not really obvious?

Reply:
Apparently it varies: https://github.com/alphagov/content-publisher#running-the-test-suite. It does kind of help to reassure people they've run all the tests, I suppose.